ui/eureka/eureka.js

```diff
@@ -18,10 +18,27 @@ let Options = {
       'Eureka Hydatos': '丰水之地',
     },
     ko: {
-      'Eureka Anemos': '아네모스 지대',
-      'Eureka Pagos': '파고스 지대',
-      'Eureka Pyros': '피로스 지대',
-      'Eureka Hydatos': '히다토스 지대',
+      'Eureka Anemos': '아네모스편',
+      'Eureka Pagos': '파고스편',
+      'Eureka Pyros': '피로스편',
+      'Eureka Hydatos': '히다토스편',
+    },
+  },
+  timeString: {
+    'for ([0-9]{1,3})m': {
+      'ko': '$1분 동안',
+    },
+    // 'for \?\?\?': {
```

Comment:
I get `Uncaught SyntaxError: Invalid regular expression: /for ???/: Nothing to repeat` error here. I don't know why this is invalid.

Reply:
It's a string, so you have to double escape if you want to , `'for \\?\\?\\?'`

Reply:
I didn't understand actually, putting `for \?\?\?` into Regexp changes to `for ???`?